### PR TITLE
Change default pattern to include thumbs up emoji

### LINKED ISSUE #190: Set Unicode flag in Regex for approvals
Problem: In Github UI when you start typing a colon you'll see suggestions for emojis. When double-clicking them, Github will replace the string from the starting colon with a proper emoji. You can still choose not to insert an emoji and then your comment is technically `:+1:`, but it will still be displayed as a thumbs up emoji — so it's relatively hard to find out what happened.

This in turn breaks all the regexes that are in place because they rely on the comment body being `:+1:` instead of `👍`.

When we enable the unicode flag `u` for the regex that matches an approval, our users *should* be able to write something like `^(:+1:|👍)$` in their Zappr config.

### PR DESCRIPTION
**Pull request by prayerslayer**
Fixes #190 

Because of Github UI's changed behavior. Unfortunately the Unicode flag for `RegExp` constructor is not yet available.

### CODE REVIEW

File: .zappr-example.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 approvals:
-  # PR needs at least 4 approvals
-  minimum: 4
+  # PR needs at least 2 approvals
+  minimum: 2
```
Comment (mfellner) [inline .zappr-example.yml]: Maybe we could also make the "users" example slightly more generic. Currently we have this:

```yaml
# this guy
users:
  - mfellner
```

Comment (prayerslayer) [inline .zappr-example.yml]: As long as it doesn't become too generic/abstract, sure. Ideas?

File: package.json [hunk 2]
```diff
@@ -44,7 +44,7 @@
     "babel-loader": "6.2.x",
     "babel-plugin-webpack-loaders": "0.4.0",
     "babel-preset-es2015": "6.6.0",
-    "babel-preset-node5": "11.0.1",
+    "babel-preset-es2015-node5": "1.2.0",
```
Comment (mfellner) [inline package.json]: We should also update the `webpack.server.js` configuration to use this plugin.

Comment (prayerslayer) [inline package.json]: Done

**Comment by mfellner** [comment]
:+1:

**Comment by mfellner** [comment]
:+1:

**Pull request merged**